tests/main/confinement-classic/task.yaml

```diff
@@ -15,10 +15,8 @@ execute: |
     run_install --classic
     /snap/bin/test-snapd-hello-classic | MATCH 'Hello Classic!'
 
-    # FIXME jailmode doesn't work on debian type systems due to missing
-    # confinement yet. This check will be replaced with proper detection
-    # if confinement is supported on the platform we're running.
-    if [[ "$SPREAD_SYSTEM" = debian-* ]]; then
+    if snap forced-devmode; then
+        echo "WARNING: Skipping confinement checks as snapd runs in forced devmode"
```

Comment:
These warnings may be dropped. They won't go anywhere useful, so it's just adding noise to the test text.